_rules/meta-viewport-b4f0c3.md

```diff
@@ -24,20 +24,26 @@ acknowledgments:
 
 This rule applies to each `content` attribute on a `meta` element with a `name` [attribute value][] of `viewport` for which at least one of the following is true:
 
-- the `content` [attribute value][] has the `user-scalable` property; or
-- the `content` [attribute value][] has the `maximum-scale` property.
+- the `content` attribute specified value includes the text `user-scalable`; or
+- the `content` attribute specified value includes the text `maximum-scale`.
+
+**Note:** The Applicability refers to string inclusion of the indicated strings in the value of the `content` attribute, as written.
 
 ## Expectation 1
 
-For each test target, the [attribute value][] does not have a `user-scalable` property with a value of `no`.
+For each test target, the [attribute value][] does not have a `user-scalable` property with a value of `fixed`.
+
+The [attribute value][] of the `content` attribute is a list of key-value pairs, obtained by parsing the specified value, then applying the [translations into `@viewport` descriptors][descriptor translation], notably [the translation for `user-scalable`](https://www.w3.org/TR/css-device-adapt-1/#user-scalable), and using the value obtained for `user-zoom`.
```

Comment:
Trying to make the algorithm clearer. Also adding the "For the purpose of this rule" at the beginning so that it is clear that the attribute value is not computed using our standard definition. 

```suggestion
For the purpose of this rule, the [attribute value][] of the `content` attribute is a list of key-value pairs, obtained by:

1. parsing the specified value for each key-value pair;
2. applying the [translations into `@viewport` descriptors][descriptor translation];
3. using the value of the corresponding `@viewport` descriptor, notably using the value obtained for the `user-zoom` descriptor from [the translation for `user-scalable`](https://www.w3.org/TR/css-device-adapt-1/#user-scalable).
```